schemas/ethereumStorage.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+module.exports = () => {
+  return new Schema({
+    chain: {type: String},
+    timeStamp: {type: Number},
+    numberOfHosts: {type: Number},
```

Comment:
Naming! numberOfHosts is called activeHosts and hashrate is called hashRate in the public chain. Fix this after the demonstration.